sa/database.go

```diff
@@ -103,7 +94,8 @@ func DBMapForTest(dbConnect string) (*boulderDB.WrappedMap, error) {
 		return nil, err
 	}
 
-	return newDbMapFromMySQLConfig(config, DbSettings{})
+	blog.NewMock()
```

Comment:
What's the purpose of this line? I don't think the blog package initializes any global state when NewMock is called.

Reply:
I think this might be a leftover from refactoring `//cmd/expiration-mailer.go`'s `setup` function.